client/src/app/login/page.tsx

```diff
@@ -35,6 +35,7 @@ export default function LoginPage() {
     const error = await login(credentials.email, credentials.password);
 
     setError(error);
+    redirect('/api/logout');
```

Comment:
We shouldn't be logging out, I don't think cookies will get set if this errors? Test it and see